.github/workflows/tests.yml

```diff
@@ -75,14 +75,15 @@ jobs:
     env:
       CGO_LDFLAGS_ALLOW: "(-Wl,-wrap,pthread_create)|(-Wl,-z,now)"
       LXD_SHIFTFS_DISABLE: "true"
+      LXD_CEPH_CLUSTER: "ceph"
     name: System tests
     runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
         go: ["1.18.x"]
         suite: ["cluster", "standalone"]
-        backend: ["dir", "btrfs", "lvm", "zfs", "random"]
+        backend: ["dir", "btrfs", "lvm", "zfs", "ceph", "random"]
```

Comment:
Dunno if that makes sense for that debug session to trim down to a single `suite` and `backend: ["ceph"]`, while in draft?